Userland/Applications/CrashReporter/main.cpp

```diff
@@ -68,6 +79,8 @@ static TitleAndText build_backtrace(Coredump::Reader const& coredump, ELF::Core:
 
     auto timer = Core::ElapsedTimer::start_new();
     Coredump::Backtrace backtrace(coredump, thread_info, [&](size_t frame_index, size_t frame_count) {
+        if (!progress_window->window_id())
+            return;
```

Comment:
Question: When does this trigger? If the above `on_close -> exit()` triggers, then this code doesn't run anymore. If it doesn't trigger, then this code doesn't need to check whether `progress_window` still exists.

If this `if` does make sense (and I just overlooked something), would a `if (!progress_window.is_visible())` make sense? It gets updated at basically the same time as `m_window_id`, and is a bit nicer to read. :bike:

Reply:
See my comment above

Reply:
Ha, a real-life race condition :D